src/components/Panels/Guider/Diagram/Diagram.tsx

```diff
@@ -225,8 +261,14 @@ function Flow() {
     // High-O
     let highoState: string;
     if (state.m1CorrectionsEnable) {
+      // Check if any source is active
+      const highoActive = sourceEdges.filter((n) => n.target === 'higho').some((n) => isSourceActive(n.id));
```

Comment:
Why not combine these into one? 
```suggestion
      const highoActive = sourceEdges.some((n) => n.target === 'higho' && isSourceActive(n.id));
```

Reply:
No reason at all, just wasn't thinking straight.